Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/NieuwsEnWerkInstructiesScenarios.cs

```diff
@@ -685,11 +685,7 @@ public async Task Scenario21()
 
         await Step("And clicks on the submit button");
 
-        var opslaanKnop = Page.GetByRole(AriaRole.Button, new() { Name = "Opslaan" });
-        while (await opslaanKnop.IsVisibleAsync() && await opslaanKnop.IsEnabledAsync())
-        {
-            await opslaanKnop.ClickAsync();
-        }
+       await using var bericht = await Page.OnSaveBericht();
```

Comment:
suggestion. instead of replacing the titel, leave the original titel and append something to it, like '_updated'. and adjust the dispose function of bericht to match on the Title or on Title+'_updated' or an item with a title that starts with 'Title'

Reply:
outdated